docs/source/kale.embed.rst

```diff
@@ -49,6 +49,24 @@ kale.embed.image\_cnn module
    :undoc-members:
    :show-inheritance:
 
+kale.embed.lenet module
```

Comment:
Is it necessary to add such a module when there is already an embed.image_cnn module?

Reply:
merged lenet module into embed.image_cnn